extensions/wikia/PhalanxII/PhalanxSpecialController.class.php

```diff
@@ -127,19 +127,19 @@ public function main() {
 			]
 		];
 
-		if (!$showEmailBlock) {
-			unset($typeSections['account-creation']);
+		if ( !$showEmailBlock ) {
+			unset( $typeSections['account-creation'] );
 		}
 
 		$this->setVal( 'expiries', $expiries );
 		$this->setVal( 'listing', $listing );
-		$this->setVal( 'data',  $data);
-		$this->setVal( 'editMode',  $editMode);
+		$this->setVal( 'data',  $data );
+		$this->setVal( 'editMode',  $editMode );
 		$this->setVal( 'action', $this->title->getLocalURL() );
 		$this->setVal( 'typeFilter', $pager->getSearchFilter() );
 		$this->setVal( 'blockTypes', $blockTypes );
-		$this->setVal( 'type', $this->wg->Request->getInt('type') );
-		$this->setVal( 'typeSections', $typeSections);
+		$this->setVal( 'type', $this->wg->Request->getInt( 'type' ) );
+		$this->setVal( 'typeSections', $typeSections );
 
 		// SUS-270: preload username into filter search box
 		$this->setVal( 'checkBlocker', $data['checkBlocker'] !== '' ? $data['checkBlocker'] : $data['text'] );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "checkBlocker" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)